app/screens/channel/channel_base.js

```diff
@@ -327,6 +329,15 @@ export default class ChannelBase extends PureComponent {
     }
 }
 
+class StyleFactory {
```

Comment:
in this case you want to use `makeStyleSheetFromTheme` from app/utils/theme as it memoizes the theme, you can see examples of this through the whole codebase.

Reply:
doh! Thanks 👍